CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.17.0] - 2023-06-26
```

Comment:
@severinoAmmirati 

Please remove this section. It will get added automatically by the release workflow.

Reply:
@neotrow Removed